src/LightArrow.jsx

```diff
@@ -246,15 +247,15 @@ export const LightArrow = function({
     <div
       className={classnames(
         styles.lightArrowContainer,
-        enabled ? styles.enabled : null,
+        enabled ? styles.enabled : styles.enabled,
         up ? styles.up : null,
         down ? styles.down : null,
         className,
       )}
       style={{
-        transform: `translate(${x}px, ${y}px)`,
-        left: ax,
-        top: ay,
+        left: `75%`,

```

Comment:
This could be in the stylesheet. Also, would prefer to use pixel values rather than percentages.